presign/base/models.py

```diff
@@ -407,11 +408,29 @@ def change_state(self, action):
         self.state = next_state
         self.save(update_fields=["state"])
 
-    def send_change_state_email(self, request, action):
+        # if the action is not a answer save, we set all current messages to inactive
+        if action != ParticipantStateActions.ANSWERS_SAVED:
+            ParticipantCustomMessage.objects.filter(
+                participant=self, is_current=True
+            ).update(is_current=False)
+
+        if custom_msg:
+            ParticipantCustomMessage.objects.create(
+                participant=self, message=custom_msg, is_current=True
+            )
```

Comment:
Maybe `state_change` should automatically try to send the email? I think all locations where the state is changed, an email should be sent as well (that already happens in all locations except one)

Reply:
I don't see what this part has to do with sending the mail? It only saves the message into the database.